src/api/AuthServer/AuthServer/Services/JwtService.cs

```diff
@@ -0,0 +1,44 @@
+﻿using AuthServer.Services.Interfaces;
+using Microsoft.IdentityModel.Tokens;
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Text;
+
+namespace AuthServer.Services
+{
+    public class JwtService : IJwtService
+    {
+        private readonly IConfiguration _configuration;
+
+        public JwtService(IConfiguration configuration)
+        {
+            _configuration = configuration;
+        }
+
+        public string GenerateJwtToken(string userId, string userName, string email)
+        {
+            var claims = new[]
+            {
+            new Claim(ClaimTypes.NameIdentifier, userId),
+            new Claim(ClaimTypes.Name, userName),
+            new Claim(ClaimTypes.Email, email)
+        };
+
+            var key = new SymmetricSecurityKey(Encoding.UTF8.GetBytes(_configuration["Jwt:SecretKey"]));
+            var credentials = new SigningCredentials(key, SecurityAlgorithms.HmacSha256);
+
+            var tokenOptions = new JwtSecurityToken(
+                issuer: _configuration["Jwt:Issuer"],
+                audience: _configuration["Jwt:Audience"],
```

Comment:
Why do we use `_configuration` here? You already added `JwtSettings` class. You can use Options pattern, register JwtSettings as `IOptions<JwtSettings>` and then use them here.